lib/llvm/src/operations.rs

```diff
@@ -805,13 +809,19 @@ fn translate_intr_libcall(
         ));
     }
 
-    let name = strings.get_extname(name);
+    let libcall = match name {
+        "memset" => ir::LibCall::Memset,
+        "memcpy" => ir::LibCall::Memcpy,
+        "memmove" => ir::LibCall::Memmove,
+        _ => panic!("not handled {}", name)
+    };
```

Comment:
`translate_intr_libcall` isn't supposed to be called for "memcpy" etc., as they should be handled by `translate_mem_instrinsic`.